src/ComplexMethod.java

```diff
@@ -8,7 +8,7 @@ public String intToEnglishValue(int i) {
             return "Two";
         }
         if (i == 3) {
-            return "Three";
+            return "three---";
         }
         if (i == 4) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS109)